### PR TITLE
Add candidates tab

### PR DESCRIPTION
**Pull request by cipick**

### CODE REVIEW

File: src/components/BallotPage.tsx [hunk 1]
```diff
@@ -64,6 +65,8 @@ const BallotContent: React.FC<{ ballotId: number; onOpenSidebar?: () => void }>
 
   const electionType = meta?.type;
   const compatibleScopes = electionType ? electionTypeCompatibleScopes(electionType) : undefined;
+  const showCandidatesTab =
```
Comment (RaduCStefanescu) [inline src/components/BallotPage.tsx:68]: can we make this available to take into consideration the fact that it is a live election?

File: src/components/CandidatesTab.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      discardPreviousData: true,
+    };
+  }, [api, ballotId, scope.type, scope.countyId]);
+
+  return (
+    <EmbedButtonWrapper>
+      {meta && meta.live && (
+        <>
+          <DivBody className={classes.processHeading}>
+            Graficele arată procentele obținute de fiecare candidat pe măsură ce sunt numărate voturile. Datele se
```
Comment (RaduCStefanescu) [inline src/components/CandidatesTab.tsx:39]: This is not needed for the candidates tab.

File: src/components/BallotPage.tsx (hunk 1 above)
Comment (cipick) [reply src/components/BallotPage.tsx]: fixed 👍

File: src/components/CandidatesTab.tsx (hunk 2 above)
Comment (cipick) [reply src/components/CandidatesTab.tsx]: 👍 removed

**Review by RaduCStefanescu — APPROVED** [review]

**Pull request merged**